src/components/NavigationTabs/index.tsx

```diff
@@ -16,11 +17,7 @@ const Tabs = styled.div`
   justify-content: space-evenly;
 `
 
-const activeClassName = 'ACTIVE'
-
-const StyledNavLink = styled(NavLink).attrs({
-  activeClassName,
-})`
+const StyledNavLink = styled(({ isActive: _, ...rest }) => <NavLink {...rest} />)<{ isActive: boolean }>`
```

Comment:
Styled components will forward all props to dom and it will create error in react-dom if its not a valid HTML attribute. So we need to remove unwanted props and dont forward it down. It was a short hand for the below. I can update to the below 

```
styled(NavLink).withConfig({
  shouldForwardProp: prop =>  prop !== 'isActive' 
})
```

https://github.com/styled-components/styled-components/issues/2878